tests/notifier/notifiers/email_factory_test.py

```diff
@@ -14,7 +14,7 @@
 
 """Tests for Email Factory"""
 
-from google.cloud.forseti.common.util.email import email_factory
+from google.cloud.forseti.common.util.email import email_factory, mailjet_connector
```

Comment:
Each import should be in a separate line.
```
from google.cloud.forseti.common.util.email import email_factory
from google.cloud.forseti.common.util.email import mailjet_connector
```

Reply:
done